src/huggingface_hub/keras_mixin.py

```diff
@@ -144,6 +151,172 @@ def _create_model_card(
+        api_endpoint (:obj:`str`, `optional`):
+            The API endpoint to use when pushing the model to the hub.
+        git_user (``str``, `optional`):
+            will override the ``git config user.name`` for committing and pushing files to the hub.
+        git_email (``str``, `optional`):
+            will override the ``git config user.email`` for committing and pushing files to the hub.
+        task_name (:obj:`str`, `optional`):
+            Name of the task the model was trained on. See the available tasks at https://github.com/huggingface/huggingface_hub/blob/main/js/src/lib/interfaces/Types.ts.
+        model_plot (:obj:`bool`):
+            Setting this to `True` will plot the model and put it in the model card. Requires graphviz and pydot to be installed.
+        include_optimizer (:obj:`bool`, `optional`):
+            Whether or not to include optimizer during serialization.
+        model_save_kwargs(:obj:`dict`, `optional`):
+            model_save_kwargs will be passed to tf.keras.models.save_model().
+    """
+
+    def __init__(
+        self,
+        save_strategy: Optional[str] = "epoch",
+        save_steps: Optional[int] = None,
+        repo_path_or_name: Optional[str] = None,
+        repo_url: Optional[str] = None,
+        hub_token: Optional[str] = None,
+        organization: Optional[str] = None,
+        api_endpoint: Optional[str] = None,
+        git_user: Optional[str] = None,
+        git_email: Optional[str] = None,
+        model_plot: Optional[bool] = True,
+        include_optimizer: Optional[bool] = True,
+        task_name: Optional[str] = None,
+    ):
+        super().__init__()
+        self.save_strategy = save_strategy
+        self.save_steps = save_steps
+        self.repo_path_or_name = repo_path_or_name
+        self.repo_url = repo_url
+        self.organization = organization
+        self.hub_token = hub_token
+        self.api_endpoint = api_endpoint
+        self.model_plot = model_plot
+        self.task_name = task_name
+        self.last_job = None
+        self.git_user = git_user
+        self.git_email = git_email
+        self.include_optimizer = include_optimizer
+        self.task_name = task_name
+
+        if self.repo_path_or_name is None and self.repo_url is None:
+            raise ValueError(
+                "You need to specify a `repo_path_or_name` or a `repo_url`."
+            )
+
+        if isinstance(hub_token, bool) and hub_token:
+            self.token = HfFolder.get_token()
+        elif isinstance(hub_token, str):
+            self.token = hub_token
+        else:
+            self.token = None
+
+        if self.token is None:
+            raise ValueError(
+                "You must login to the Hugging Face hub on this computer by typing `huggingface-cli login` and "
+                "entering your credentials to use `use_auth_token=True`. Alternatively, you can pass your own "
+                "token as the `use_auth_token` argument."
```

Comment:
This does not hold with my suggestion above, so you need to change it a bit. Also, there is no `use_auth_token` argument